tests/cli/test_cli_core.py

```diff
@@ -457,11 +481,20 @@ def test_scan(
     cli_runner: CliRunner,
     trailblazer_context: Dict[str, MockStore],
     caplog,
+    mocker,
     ongoing_analysis_case_name: str,
+    slurm_squeue_output: Dict[str, Path],
 ):
     """Test scanning for analyses and updating analysis status."""
     caplog.set_level("INFO")
 
+    mocker.patch(
+        FUNC_GET_SLURM_SQUEUE_OUTPUT_PATH,
+        return_value=subprocess.check_output(
+            ["cat", slurm_squeue_output.get(ongoing_analysis_case_name)]
+        ).decode("utf-8"),
+    )
+
```

Comment:
I see this mocker repeated in your tests, is it possible to make a fixture of it? I'm not sure about this suggestion feel free to disregard

Reply:
I think it is best to use it in test. The  slurm_squeue_output is already a fixture and needs dynamic input from the actual test.